apps/frontend/app/(website)/studio/src/store/snippets.ts

```diff
@@ -108,174 +107,186 @@ const getEditorsFromLS = () => {
   if (!editors) return;
   return;
 };
-export const useSnippetsStore = create<SnippetsState>(
-  persist(
-    (set, get) => ({
-      tabsLimit: 12,
-      getHasReachedTabsLimit: () => get().editors.length >= get().tabsLimit,
-      editors: INITIAL_STATE.editors,
-      addPair: () => {
+export const useSnippetsStore = create<SnippetsState>((set, get) => ({
+  tabsLimit: 12,
+  getHasReachedTabsLimit: () => get().editors.length >= get().tabsLimit,
+  // editors: localStorage.getItem('editors') ? JSON.parse(localStorage.getItem('editors')) : [getSingleTestCase()],
+  addPair: (name?: string) => {
+    set({
+      editors: [
+        ...get().editors,
+        {
+          name:
+            name ||
+            `Test ${
+              (get()
+                .getAllNames()
+                .filter((name) => name.toLowerCase().startsWith("test "))
+                .map((name) => name.split(" ")[1])
+                .map(Number)
+                .filter(Boolean)
+                .at(-1) || 0) + 1
+            }`,
+          before: getSnippetInitialState(),
+          after: getSnippetInitialState(),
+          output: getSnippetInitialState(),
+        },
+      ],
+    });
+    setTimeout(
+      () =>
+        set({
+          selectedPairIndex: get().editors.length - 1,
+        }),
+      100,
+    );
+  },
+  editors: [],
+  renameEditor: (index) => (name) => {
+    const obj = get();
+    obj.editors[index].name = name;
+    set(obj);
+  },
+  removePair: (index: number) => {
+    const editors =
+      get().editors.length > 1
+        ? remove(index, 1, get().editors)
+        : get().editors;
+    if (index === get().selectedPairIndex) {
+      set({
+        selectedPairIndex: 0,
+        editors,
+      });
+    } else
+      set({
+        selectedPairIndex: Math.max(get().selectedPairIndex - 1, 0),
+        editors,
+      });
+  },
```

Comment:
spaces and early returns please

Reply:
this one has not been touched in this PR